client/components/LearnerViewDirective.js

```diff
@@ -381,6 +397,7 @@ tie.directive('learnerView', [function() {
         for (var i = 0; i < $scope.questionIds.length; i++) {
           $scope.questionsCompletionStatus.push(false);
         }
+        $scope.displayAutoSaveText = false;
```

Comment:
Let's go with `$scope.autosaveTextIsDisplayed = false;`, so that this does not start with a verb (which makes it seem like an action).